pages/apply.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html><head>
+<link rel="shortcut icon" href="https://ssl.gstatic.com/docs/spreadsheets/forms/favicon_qp2.png" type="image/x-icon">
+<meta http-equiv="Content-type" content="text/html; charset=utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=10; chrome=1;">
+<meta name="fragment" content="!">
+<meta name="viewport" content="width=device-width">
+<base target="_blank">
+<title>Apply to Join 18F</title>
+<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:400,700">
+
+<link rel="stylesheet" type="text/css" href="../assets/css/all.css">
+<link rel="stylesheet" type="text/css" href="../assets/css/form.css">
+
+<script type="text/javascript" src="../assets/js/jquery-1.11.3.min.js"></script>
+<script type="text/javascript" src="../assets/js/parsley.min.js"></script>
+
+<body>
+
+  <div id="main">
+
+    <header>
+      <h1>Apply to Join 18F</h1>
+      <img src="../assets/img/18f-logo.png" alt="18F Logo" class="logo" />
+    </header>
+
+    <script type="text/javascript">var submitted=false;</script>
+    <iframe name="hidden_iframe" id="hidden_iframe"
+    style="display:none;" onload="if(submitted) {
+      $(document).ready(function() {
+        window.location.href = './apply-thank-you.html';
+      })}"></iframe>
+
+    <form action="https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSfbe4KAEqQFp07f8zfD6xWAfdFTAaZ04tpDQRJnNPeBHoW6RQ/formResponse" method="post"
+    target="hidden_iframe" onsubmit="submitted=true;">
+
+      Fields that are marked with <span class="required">*</span> are required.
+
+      <fieldset class="eligibility">
+
+        <h2><legend>Eligibility:<span class="required">*</span></legend></h2>
+        <p>Any individual who is not currently a GSA employee or contractor and who is a U.S. citizen, a non-citizen national of the U.S. or holds a valid green card is eligible to apply.</p>
+
+        <div class="usa-input">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input type="radio" name="entry.1377828897" value="Verify that you meet the above eligibility requirements" id="group_1377828897_1" role="radio" class="ss-q-radio" aria-required="true">
```

Comment:
In case you're curious, the display issue was due to not encapsulating the options in `<ul>` (unordered list) and `<li>` (list item).